grails-app/services/org/pih/warehouse/data/PersonDataService.groovy

```diff
@@ -71,7 +71,7 @@ class PersonDataService {
     }
 
     String[] extractNames(String combinedNames) {
-        String[] names = combinedNames.split(" ", 2)
+        String[] names = combinedNames.split(" ", 2)*.trim()
```

Comment:
I like the spread operator, but feel a little weird using it when we're mutating the results. It's probably better to use a collect to be more readable and explicit.
```
String[] names = combinedNames.split(" ", 2).collect { name?.trim() }
```

With that said, the official Groovy docs seem to imply (using the phrase "invoke an action on all items") so I'm okay with using it this way as long as everyone else is okay with it. Since @awalkowiak is the only other reviewer, I'll let him voice his opinion.

> The Spread-dot Operator (*.), often abbreviated to just Spread Operator, is used to invoke an action on all items of an aggregate object. 

Source: https://groovy-lang.org/operators.html#_spread_operator